Tests failing

The tests are failing on Travis CI and locally:

```
====================================================== FAILURES =======================================================
_________________________________ TestRecordOnce.test_replays_response_from_cassette __________________________________

self = <tests.integration.test_record_modes.TestRecordOnce testMethod=test_replays_response_from_cassette>

    def test_replays_response_from_cassette(self):
        s = self.session
        with Betamax(s).use_cassette('test_replays_response') as betamax:
            self.cassette_path = betamax.current_cassette.cassette_path
            assert betamax.current_cassette.is_empty() is True
            r0 = s.get('http://httpbin.org/get')
            assert r0.status_code == 200
            assert betamax.current_cassette.interactions != []
            assert len(betamax.current_cassette.interactions) == 1
            r1 = s.get('http://httpbin.org/get')
            assert len(betamax.current_cassette.interactions) == 2
            assert r1.status_code == 200
            r0_headers = r0.headers.copy()
            r0_headers.pop('Date')
            r0_headers.pop('Age', None)
            r1_headers = r1.headers.copy()
            r1_headers.pop('Date')
            r1_headers.pop('Age', None)
            # NOTE(sigmavirus24): This fails if the second request is
            # technically a second later. Ignoring the Date headers allows
            # this test to succeed.
>           assert r0_headers == r1_headers
E           AssertionError: assert {'Server': 'm...-Origin': '*'} == {'Server': 'me...-Origin': '*'}
E             Full diff:
E             - {'Server': 'meinheld/0.6.1', 'Content-Length': '267', 'Via': '1.1 vegur', 'Access-Control-Allow-Credentials': 'true', 'X-Processed-Time': '0.0011579990387', 'Connection': 'keep-alive', 'X-Powered-By': 'Flask', 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*'}
E             ?                                                                                                                                                ^^  ^^^^^^^^^^^^^^^^^^^^^^^
E             + {'Server': 'meinheld/0.6.1', 'Content-Length': '267', 'Via': '1.1 vegur', 'Access-Control-Allow-Credentials'...
E             
E             ...Full output truncated (2 lines hidden), use '-vv' to show

tests/integration/test_record_modes.py:38: AssertionError
```

I'll have a look at this first so we can rely on Travis CI once again.

--- a/.travis.yml
+++ b/.travis.yml
@@ -29,6 +29,10 @@ matrix:
       env: TOXENV=py34 REQUESTS_VERSION="===2.2.1"
     - python: 3.5
       env: TOXENV=py35 REQUESTS_VERSION="===2.2.1"
+    - python: 3.6
+      env: TOXENV=py36 REQUESTS_VERSION="===2.2.1"
+    - python: pypy
+      env: TOXENV=pypy REQUESTS_VERSION="===2.2.1"
     - env: TOXENV=py27-flake8
     - env: TOXENV=py34-flake8
     - env: TOXENV=docstrings

--- a/tests/integration/test_record_modes.py
+++ b/tests/integration/test_record_modes.py
@@ -29,12 +29,16 @@ class TestRecordOnce(IntegrationHelper):
             r0_headers = r0.headers.copy()
             r0_headers.pop('Date')
             r0_headers.pop('Age', None)
+            r0_headers.pop('X-Processed-Time', None)
             r1_headers = r1.headers.copy()
             r1_headers.pop('Date')
             r1_headers.pop('Age', None)
+            r1_headers.pop('X-Processed-Time', None)
             # NOTE(sigmavirus24): This fails if the second request is
             # technically a second later. Ignoring the Date headers allows
             # this test to succeed.
+            # NOTE(hroncok): httpbin.org added X-Processed-Time header that
+            # can possibly differ (and often does)
             assert r0_headers == r1_headers
             assert r0.content == r1.content
 

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py32,py33,py34,pypy,{py27,py34}-flake8,docstrings
+envlist = py27,py33,py34,py35,py36,pypy,{py27,py34}-flake8,docstrings
 
 [testenv]
 pip_pre = False

